operators/api-key-sealedsecret/api-key-sealedsecret.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Looks good, but this is the wrong location. It should be in the services/instances (this is not an operator but an instance of the sealed secret operator which was already created)

Don't forget to make an app in otp-gitops too!

Reply:
Moved: 
```
 renamed:    operators/api-key-sealedsecret/api-key-sealedsecret.sh -> instances/api-key-sealedsecret/api-key-sealedsecret.sh
 renamed:    operators/api-key-sealedsecret/api-key-secret.yaml -> instances/api-key-sealedsecret/api-key-secret.yaml
```

Reply:
Added the app in otp-gitops: https://github.com/oto-cp4ba-gitops/otp-gitops/pull/14

not sure if that's correct, however.

Reply:
Correct.

I added the Kustomize entry too. Thanks and well done